sdk/spring/azure-spring-boot-starter-cosmos/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 3.12.0-beta.1 (Unreleased)
+This release is compatible with Spring Boot 2.5.5-2.5.8, 2.6.0-2.6.1.
```

Comment:
Why the libraries are not working with the lower version of Spring Boot 2.5.x?

Reply:
Due to netty break change `java.lang.NoSuchMethodError: io.netty.internal.tcnative.SSLContext.setCurvesList(J[Ljava/lang/String;)Z`, eventhubs and cosmos sample will not work in Spring Boot 2.5.4 version.

Reply:
Could you please offer more details here? Like EH or SB is using which netty version and such netty version is brought by which azure-core lib? And this SSLContext.setCurvesList(J[Ljava/lang/String;)Z is added in which netty version?

Reply:
LIst the following dependencies according to the dependency of the error message:
- azure-messaging-eventhubs:5.10.3 -> azure-core:1.22.0->netty-tcnative-boringssl-static:2.0.43.Final
- spring booot 2.5.5 -> netty-tcnative:2.0.43.Final
- spring booot 2.5.4 -> netty-tcnative:2.0.40.Final, currently found exception in Cosmos, Eventhusb samples